Examples/Framework/include/ActsExamples/Framework/Sequencer.hpp

```diff
@@ -78,6 +78,9 @@ class Sequencer {
     std::string outputDir;
     /// output name of the timing file
     std::string outputTimingFile = "timing.tsv";
+    /// Output name of the per event timing file
+    /// Enable detailed per event timing writing
+    std::optional<std::string> outputEventTimingFile = std::nullopt;
```

Comment:
```suggestion
    std::string outputEventTimingFile = "timing-events.tsv;
```

I think a string is sufficient. We usually check if it is empty to see if the output is wanted or not.

I would also argue that the new output can be useful by default and does not harm anyone.

Reply:
In the issue, @paulgessinger thought that it would be better off by default as it might produce very large files (on row for each algorithm for each event).
Ref: [issue  #1700](https://github.com/acts-project/acts/issues/1700#issuecomment-1330332667)

Reply:
I would still remove the optional

Reply:
Done